x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/google_workspace_admin_role_deletion.json

```diff
@@ -33,5 +33,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 8
+  "version": 10
```

Comment:
Looks to be unnecessary, _only version_ was updated in the last `8.2 rules` PR from `6` to `8`: https://github.com/elastic/kibana/pull/129526/files#diff-64ac778ef177869cd5d65f7e04967406ba6325df71188c4e999fab132cc69813R36

Reply:
Double bump is the result of manual bumps done from the following.

https://github.com/elastic/detection-rules/pull/1870/files